### PR TITLE
Default layout

### PR DESCRIPTION
**Pull request by hoangtuNhat1**
Related Tickets
[#82874](https://edu-redmine.sun-asterisk.vn/issues/82874)
WHAT (optional)
Làm UI chung cho all role bao gồm: Home, Contact, Navabar
HOW
Create view.controller static pages, add i18n 
Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

**Comment by hoangtuNhat1** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons/font/bootstrap-icons.css" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js"></script>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: Không dùng cdn nhé
Dùng import map nha

File: app/views/static_pages/contact.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<%= provide :title, t("view.static_pages.contact.title") %>
+<div class="container mt-5">
+  <div class="text-center mb-4">
+    <h1 class="display-5 fw-bold text-dark"><%= t('view.static_pages.contact.title') %></h1>
+  </div>
+  <div class="list-group">
+    <!-- Địa chỉ -->
+    <div class="list-group-item d-flex align-items-center">
+      <i class="bi bi-geo-alt fs-4 me-3 text-dark"></i>
+      <span class="fs-5 text-dark"><%= t('view.static_pages.contact.address') %></span>
```
Comment (ghost) [inline app/views/static_pages/contact.html.erb]: Dùng "

**Review by ghost — COMMENTED** [review]
cmt

**Comment by hoangtuNhat1** [comment]
ready

File: app/views/static_pages/_slider.html.erb [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    <div class="carousel-item">
+      <img src="<%= asset_path "slide2.jpg" %>" class="d-block" alt="Slide 2">
+      <div class="carousel-caption d-none d-md-block">
+        <h5><%= t "view.static_pages.home.slide.service_quality" %></h5>
+        <p><%= t "view.static_pages.home.slide.service_commitment" %></p>
+      </div>
+    </div>
+
+    <div class="carousel-item">
+      <img src="<%= asset_path "slide3.jpg" %>" class="d-block" alt="Slide 3">
```
Comment (ghost) [inline app/views/static_pages/_slider.html.erb]: Dùng image_tag đi em

**Review by ghost — COMMENTED** [review]
cmt

**Comment by hoangtuNhat1** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**